config/settings/test.py

```diff
@@ -47,3 +47,5 @@
         "level": "DEBUG",
     }
 }
+
+EVENTS_QUEUE_ASYNC_CONNECTION = False
```

Comment:
Needed to force to use blockingconnection when testing as we can test with geventconnection